src/openforms/authentication/contrib/yivi_oidc/admin.py

```diff
@@ -1,17 +1,46 @@
 from django.contrib import admin
 
+from import_export import resources
+from import_export.admin import ImportExportModelAdmin
+
 from .models import AttributeGroup
 
 
+class AttributeGroupResource(resources.ModelResource):
+    class Meta:
+        model = AttributeGroup
+        # Use uuid as identifier
+        import_id_fields = ("uuid",)
+        fields = ("name", "uuid", "description", "attributes")
+
+    def dehydrate_name(self, obj: AttributeGroup) -> str:
+        # Trimming whitespace
+        return obj.name.strip()
+
+    def dehydrate_description(self, obj: AttributeGroup) -> str:
+        # Trimming whitespace
+        return obj.description.strip()
+
+
 @admin.register(AttributeGroup)
-class AttributeGroupAdmin(admin.ModelAdmin):
+class AttributeGroupAdmin(ImportExportModelAdmin):
+    import_template_name = "admin/forms/attributegroup/import_form.html"
+    export_template_name = "admin/forms/attributegroup/export_form.html"
+
     fields = [
         "name",
+        "uuid",
         "description",
         "attributes",
     ]
+    readonly_fields = ["uuid"]
```

Comment:
```suggestion
    readonly_fields = ("uuid",)
```
I prefer tuples at the class/module level and especially in model admins because they're immutable and better convey the intent that this is not supposed to be modified at runtime.